### PR TITLE
Fix letter checking logic

### PR DESCRIPTION
**Pull request by octokatherine**
Hey, awesome implementation of the game!

Since I already spent some time with the game logic, I thought I'd open a pr with the fix for the issue mentioned in [this tweet](https://twitter.com/jen___sen/status/1480272136451338245?s=20)

Tested the logic locally, it's also the same way I implemented it on [word master](https://github.com/octokatherine/word-master)

Feel free to take it or not 😊

### CODE REVIEW

**Comment by octokatherine** [comment]
ah don't merge, 1 more fix coming

**Comment by rauchg** [comment]
Amazing. Thank you so much!

File: pages/_middleware.ts [hunk 1]
```diff
@@ -38,14 +38,27 @@ export default async function middleware(req : NextRequest) : Promise<NextRespon
...
+    ))
+    word.split("").forEach((letter, i) => {
+      if (WORD[i] === letter) {
+        match[i].score = "good"
+        lettersToCheck.splice(i, 1)
+      }
+    })
+    word.split("").forEach((letter, i) => {
+      if (lettersToCheck.includes(letter) && match[i].score !== "good") {
+        match[i].score = "off"
```
Comment (octokatherine) [inline pages/_middleware.ts:56]:
```suggestion
        match[i].score = "off"
        lettersToCheck.splice(lettersToCheck.indexOf(letter), 1)
```

Comment (octokatherine) [reply pages/_middleware.ts:56]: @rauchg this line needs to be added for that fix to work, sorry didn't catch it in time. want me to open another pr or do you want to commit it directly?

**Pull request merged**